### DISCUSSION TITLE
Multi-port-forwarder for multiple OSC Software.

### DISCUSSION THREAD

**Discussion by songminwoo1** [vrchat-community/osc#93] [Show and tell]
I noticed that if there are multiple OSC software listening in the same 9001 port, packets are randomly received. If one process receives one packet, others don't know if it arrived. So I made a multi-port forwarding OSC Software. I didn't decided to turn it into an open-source project yet, but I'm providing the .msi package for free. I hope to get feedbacks about it. Thank you.
https://booth.pm/ko/items/3687929

**Comment by momo-the-monster** [comment]
@songminwoo1 - thanks for sharing your contribution. We're limiting our [resources listing](https://github.com/vrchat-community/osc/wiki/Resource) to projects that are open-source or well-known so let us know if you release the source at some point and we'd be happy to revisit it!

**Reply by songminwoo1** [reply]
Ok. Cool. I just opened my source with CC BY-NC-SA 2.0 license. It's on my github repo.
https://github.com/songminwoo1/OSCConfigManager/tree/main
Thanks for your attention
